../../libiberty/strsignal.c:552: error: conflicting types for 'psignal'
/usr/include/signal.h:141: error: previous declaration of 'psignal' was here

--- gdb-6.8/libiberty/strsignal.c-orig	2007-01-31 00:13:04.000000000 +0100
+++ gdb-6.8/libiberty/strsignal.c	2008-06-17 16:30:13.000000000 +0200
@@ -549,7 +549,7 @@ followed by a newline.
 #ifndef HAVE_PSIGNAL
 
 void
-psignal (int signo, char *message)
+psignal (int signo, const char *message)
 {
   if (signal_names == NULL)
     {
